force-app/main/default/classes/OpenTasksCountTriggerHandler.cls

```diff
@@ -0,0 +1,119 @@
+/**
+ * @description       :
+ * @author            : Ferdi AKIN
+ * @group             :
+ * @last modified on  : 02-23-2023
+ * @last modified by  : Ferdi AKIN
+ **/
+public with sharing class OpenTasksCountTriggerHandler {
+  static Set<Id> accIdsSet = new Set<Id>();
+  static ID recordTypeAccount = Schema.SObjectType.Account.getRecordTypeInfosByName()
+    .get('United States')
+    .getRecordTypeId();
+
+  public static void handlerAfterInsert(List<Task> newTasks) {
+    for (Task ts : newTasks) {
```

Comment:
The code for all handlers is mostly the same and should be extracted into methods

eg.

```
public static void handleAfterUpdate(
    List<Task> oldTasks,
    List<Task> newTasks
  ) {
    Set<Id> whatIds = filterTasks(newTasks);
    whatIds.addAll(filterTasks(oldTasks));
    if (whatIds.isEmpty()) {
      return;
    }
    List<Account> accountsToUpdate = setTaskCountValues(
      getRelatedAccountsWithOpenTasks(whatIds)
    );
    update accountsToUpdate;
  }